pkg/services/rendering/plugin_mode.go

```diff
@@ -35,7 +44,46 @@ func (rs *RenderingService) renderViaPlugin(ctx context.Context, renderKey strin
 	}
 	rs.log.Debug("calling renderer plugin", "req", req)
 
-	rsp, err := rs.pluginInfo.GrpcPlugin.Render(ctx, req)
+	rsp, err := rs.pluginInfo.GrpcPluginV1.Render(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	if rsp.Error != "" {
+		return nil, fmt.Errorf("Rendering failed: %v", rsp.Error)
```

Comment:
[Error strings should not be capitalized](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)
```suggestion
		return nil, fmt.Errorf("rendering failed: %v", rsp.Error)
```